build/cluster-operator-ansible/playbooks/cluster-operator/aws/install_masters.yml

```diff
@@ -0,0 +1,49 @@
+---
+# call openshift-ansible's master provisioning
+- import_playbook: ../../aws/openshift-cluster/install.yml
+
+#- name: setup master ansible hosts
```

Comment:
if not needed, I'd delete this commented section

Reply:
the commented out piece is useful for the day when we want a standalone fetch-kubeconfig-from-target-cluster, so i've added comments to make clear why we would want to run this piece one day